dev-util/stripe-mock/metadata.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <!-- maintainer-needed -->
```

Comment:
Nope, that's not gonna work. The first rule is, never add a package with no maintainer.

Reply:
@cedk are you will to maintain stripe-mock?

Reply:
No, I have no interest in it.

Reply:
If this is going to be a blocker, I am willing to be a maintainer for this package.

stripe-mock is planning to add statefulness, at which point I want to use the package for testing myself.